manifest/Manifest.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
oh, can you please also rename this file "link-rel-manifest.https.html"... 

we might have found our first requirement/bug: what should be returned in insecure contexts? We can figure that out over in the spec.

Reply:
Filed an issue: https://github.com/w3c/manifest/issues/887